### PR TITLE
Ticker Update

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Update the banner and Epic ticker to align with the most recent design in Source. [Figma deisgn](https://www.figma.com/design/mkyzlXSnCGEOIRNncwziDC/Progress-bar-test?node-id=1-893&m=dev)

Please note that the Epic ticker colours are currently hard-coded and not configurable in the RRCP. However, the banner ticker colours can be customised via the RRCP, as this is the existing behaviour.

This has been tested end to end on CODE by creating a new banner design and epic test and ensuring that both tests render on a DCR article.

## Screenshots
New ticker:
image

On Epic:
image

image

On banner:
image

### CODE REVIEW

File: dotcom-rendering/src/components/marketing/banners/designableBanner/settings.ts [hunk 1]
```diff
@@ -33,10 +33,13 @@ export interface HighlightedTextSettings {
 }
 
 export interface TickerStylingSettings {
-	textColour: string;
+	text: string; //deprecated
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/settings.ts:36]: can this model (`TickerStylingSettings`) be deleted now that we're using the source component?

File: dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx [hunk 2]
```diff
@@ -114,6 +115,16 @@ const imageStyles = css`
 	object-fit: cover;
 `;
 
+const defaultTickerStylingSettings: TickerStylingSettings = {
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx:118]: can this use the source model now?
https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-development-kitchen/src/react-components/ticker/Ticker.tsx#L21

File: dotcom-rendering/src/components/marketing/banners/designableBanner/settings.ts (hunk 1 above)
Comment (charleycampbell) [reply dotcom-rendering/src/components/marketing/banners/designableBanner/settings.ts]: great idea - I've reworked it to do so

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx [hunk 3]
```diff
@@ -264,12 +264,15 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
 		imageSettings,
 		bannerId: 'designable-banner',
 		tickerStylingSettings: {
-			textColour: hexColourToString(ticker.text),
+			//text: hexColourToString(ticker.text),
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx:267]: some tidy up to do here

File: dotcom-rendering/src/components/marketing/hooks/useTicker.ts [hunk 4]
```diff
@@ -1,12 +1,7 @@
-/**
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/hooks/useTicker.ts:1]: can this file be removed?

Comment (charleycampbell) [reply dotcom-rendering/src/components/marketing/hooks/useTicker.ts:1]: yes good spot it's not being used anymore

File: dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx [hunk 5]
```diff
@@ -114,6 +115,16 @@ const imageStyles = css`
...
 `;
 
+const defaultTickerStylingSettings: TickerSettings['tickerStylingSettings'] = {
+	filledProgressColour: '#5056F5',
+	progressBarBackgroundColour: 'rgba(80, 86, 245, 0.35)',
+	headlineColour: '#000000',
+	totalColour: '#5056F5',
+	goalColour: '#000000',
+	//text: '#000000', // Deprecated, still needs a value
+	//goalMarker: '#000000', // Deprecated, still needs a value
```
[the remark this answers is not in the document]
Comment (charleycampbell) [reply dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx]: nice catch - complete oversight by me!!

File: dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx (hunk 3 above)
Comment (charleycampbell) [reply dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBanner.tsx]: nice catch - complete oversight by me!!

File: dotcom-rendering/src/components/ArticleMeta.apps.tsx [hunk 6]
```diff
@@ -105,7 +105,7 @@ const MetaGridByline = ({
 				padding-bottom: 0px;
 			}
 
-			address > div > span > svg:first-of-type {
+			address > div > span > div:first-of-type {
```
[the remark this answers is not in the document]
Comment (tomrf1) [reply dotcom-rendering/src/components/ArticleMeta.apps.tsx:108]: oh sorry, just noticed @domlander pushed this as a fix

[the remark this answers is not in the document]
Comment (domlander) [reply dotcom-rendering/src/components/ArticleMeta.apps.tsx:108]: This change isn't related to the ticker, but is required for the update to `source-development-kitchen`. I thought we could get this PR out with this small fix, but it turns out we can't update this without also updating `source` at the same time, which will take a bit more effort.

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**